plrust/src/user_crate/mod.rs

```diff
@@ -456,30 +456,8 @@ mod tests {
 
             let generated_cargo_toml = generated.cargo_toml()?;
             let version_feature = format!("pgx/pg{}", pgx::pg_sys::get_pg_major_version_num());
-            let fixture_cargo_toml = toml! {
-                [package]
-                edition = "2021"
-                name = crate_name
-                version = "0.0.0"
-
-                [features]
-                default = [version_feature]
-
-                [lib]
-                crate-type = ["cdylib"]
-
-                [dependencies]
-                pgx = { git = "https://github.com/tcdi/pgx", branch = "develop" }
-                // pallocator = { version = "0.1.0", git = "https://github.com/tcdi/postgrestd", branch = "1.61" }
-                /* User deps added here */
-
-                [profile.release]
-                debug-assertions = true
-                codegen-units = 1_usize
-                lto = "fat"
-                opt-level = 3_usize
-                panic = "unwind"
-            };
+            let fixture_cargo_toml = cargo_toml_template(&crate_name, &version_feature);
```

Comment:
This seems like a reasonable answer, yeah. I kind of have grown to appreciate how the tests can be annoyingly, nitpickingly exacting, especially when modifying how the crate expands into Rust code, but this particular repetition is fruitless.